.github/workflows/ci.yaml

```diff
@@ -18,6 +18,11 @@ jobs:
       - run: node --version
       - run: npm install
       - run: npm test
+      - name: coverage
+        uses: codecov/codecov-action@v1
+        with:
+          name: actions ${{ matrix.node }}
```

Comment:
nit: to be consistent, I think we can use both the OS and node version as the name here. Same below.

Reply:
Added "linux" here. Keeping names consistent with test names.

Reply:
SGTM.